### ISSUE TITLE
pbcopy error message even though xclip is installed

### ISSUE THREAD

**Issue by edrozenberg** [richrd/suplemon#252]
The file `modules/system_clipboard.py` runs a check for xsel, pbcopy, xclip in that order. As a result, even after installing `xclip`, suplemon still prints the following at each startup:

```
which: no pbcopy in (/usr/local/sbin:/usr/sbin:/sbin:/usr/local/bin:/usr/bin:/bin:/usr/games:/usr/lib/qt/bin)
```

**Comment by richrd** [comment]
I can't seem to reproduce this right now. What OS are you on?

**Comment by edrozenberg** [comment]
This was on Slackware 14.2+ Raspberry Pi 3B+.

I was trying to replicate just now on a similar machine (I gave the other one away) but after `pip3 install -U suplmeon` (`afterpygments-2.4.2 suplemon-0.2.0 wcwidth-0.1.7`) I can launch suplemon and type text but none of the other keys work (Enter, arrows, none of the ctrl- sequences etc). Can't even quit the app with any CTRL- combo (had to force quit terminal).

Closing for now, will report if I use the program and run into this again.
